### ISSUE TITLE
Can't open lws file

### ISSUE THREAD

**Issue by LegalizeAdulthood** [assimp/assimp#619]
This [Starfury model](http://www.gkane.de/files/starfury_version2_lwo.zip) from [the web](http://www.gkane.de/3dfiles/starfury_v2.html) contains an LWS which gives a parsing error when reading.  The model opens correctly in LightWave.

**Comment by kimkulling** [comment]
The archive is password-protected.

**Comment by LegalizeAdulthood** [comment]
Sorry, they were being lame.  The password is "Earthforce"

**Comment by kimkulling** [comment]
Thanks, one issue os a not supported texture format.

**Comment by LegalizeAdulthood** [comment]
Yeah, when my friend loaded it in LightWave, I noticed they were using AIFF textures.  We should still be able to import the geometry, however.

**Comment by tellypresence** [comment]
The component .lwo files seem to load OK, but getting a parse error when attempting to load the .lws file
```
Cannot parse string "EndBehavior" as a real number: does not start with digit or decimal point followed by digit.
```

**Comment by tellypresence** [comment]
That LWS file is version 1, the bundled test models look like version 3 or 5, so maybe this file is too old for the importer?

**Comment by tellypresence** [comment]
Some possibly useful references
[LightWave Scene File Format (LightWave v.4.0  June 1995)](http://www.martinreddy.net/gfx/3d/LWSC.txt): covers LWS v1

**Comment by tellypresence** [comment]
Seems like this will require expanding LWS support back to old v1

**Comment by tellypresence** [comment]
Closing for now, please reopen if someone volunteers to extend LWS support to old version 1
